tests/test_pool.py

```diff
@@ -17,630 +17,12 @@
 #
 
 
-import asyncio
-import inspect
-import random
-
 import edgedb
 
-from edgedb import compat
 from edgedb import _testbase as tb
-from edgedb import asyncio_con
-from edgedb import asyncio_pool
-from edgedb import errors
 
 
 class TestClient(tb.AsyncQueryTestCase):
-    def create_client(self, **kwargs):
```

Comment:
I think not all these tests should be removed in `test_pool.py` - I'll fix them and create a separate PR to add them back.

Reply:
OK!